tests/phpunit/class-wc-stripe-test.php

```
@@ -1000,6 +1000,43 @@ public function test_collaborators_not_reinstantiated_on_second_construction():
+	 * gateway constructor (display_paid_by_customer_amount).
+	 *
+	 * @return void
+	 */
+	public function test_gateway_settings_update_does_not_duplicate_gateway_hooks(): void {
+		WC_Stripe::get_instance()->get_main_stripe_gateway();
+		$before = $this->count_hook_callbacks( 'woocommerce_admin_order_totals_after_total', 5 );
+
+		WC_Stripe::get_instance()->gateway_settings_update( [ 'enabled' => 'no' ], false );
+
+		$this->assertSame( $before, $this->count_hook_callbacks( 'woocommerce_admin_order_totals_after_total', 5 ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_gateway_settings_update_does_not_duplicate_gateway_hooks is not named in camelCase. (undefined)

(CamelCaseMethodName)